packages/block-library/src/post-excerpt/edit.js

```diff
@@ -168,17 +167,19 @@ export default function PostExcerptEditor( {
 			value={
 				isSelected
 					? rawOrRenderedExcerpt
-					: ( trimmedExcerpt !== '...' ? trimmedExcerpt : '' ) ||
+					: ( ! isTrimmed
```

Comment:
If the text is not truncated (trimmed), we can show the original excerpt. 
Otherwise show the shorter text and the ellipsis.